src/main/java/us/talabrek/ultimateskyblock/uSkyBlock.java

```diff
@@ -276,7 +278,7 @@ public void unloadPlayerFiles() {
     public void registerEvents(PlayerDB playerDB) {
         final PluginManager manager = this.getServer().getPluginManager();
         manager.registerEvents(new PlayerNameChangeListener(this), this);
-        manager.registerEvents(new PlayerNameChangeManager(this, playerDB), this);
+        manager.registerEvents(this.playerNameChangeManager = new PlayerNameChangeManager(this, playerDB), this);

```

Comment:
This is bad coding practice.
For Java.
Assign it on a separate line instead of inline.